mbtiles/compat.py

```diff
@@ -0,0 +1,12 @@
+import sys
+import warnings
+
+if sys.version_info < (3,):
```

Comment:
I don't follow why a second warning is needed here, in addition to the one in `__init__`.

Reply:
I removed the one in `__init__` as that seemed more superfluous.